quarkus/admin/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ dependencies {
   implementation(project(":polaris-api-iceberg-service"))
 
   runtimeOnly(project(":polaris-eclipselink"))
+  runtimeOnly(project(":polaris-relational-jdbc"))
```

Comment:
This dep looks premature. Le's add it when JDBC persistence is functional and can be integration-tested inside the admin CLI.